pkg/search/scoped/context.go

```diff
@@ -102,8 +103,14 @@ func GetQueryForAllScopes(ctx context.Context) (*v1.Query, error) {
 			return nil, err
 		}
 		idField := schema.ID()
-		conjuncts = append(conjuncts, searchPkg.NewQueryBuilder().
-			AddExactMatches(searchPkg.FieldLabel(idField.Search.FieldName), scope.ID).ProtoQuery())
+		if scope.ID != "" {
```

Comment:
just confirming this isn't supposed to be something like: only one of ID and IDs should ever be populated. Right?